src/main/java/CLI/Main.java

```diff
@@ -375,4 +387,91 @@ private static void checkHistoryAddJobsAndDownload(Job job, boolean removeInputB
             downloader.run();
         }
     }
-}
+
+    public static boolean checkUpdate() throws URISyntaxException {
```

Comment:
@kmr-rohit Please change the method name to a more logical name. This method returns a boolean value indicating if any update is there. So, the method name should be `isUpdateAvailable()`.
```suggestion
    public static boolean isUpdateAvailable() throws URISyntaxException {
```

Reply:
Changed method name from checkUpdate() to isUpdateAvailable()

Reply:
Okay :+1: